pyproject.toml

```diff
@@ -67,6 +67,21 @@ features = [
   "ci",
 ]
 
+[tool.ruff]
+# smallest version ruff seems to support
+line-length = 99
+target-version = "py37"
```

Comment:
We don't normally use this, we normally use:
```toml
[project]
requires-python = ">=3.8"
```
I'm guessing this is only needed because we still support 3.6? Wonder if we could drop that soon.